angularjs-portal-frame/src/main/webapp/partials/sidebar-left.html

```diff
@@ -1,6 +1,13 @@
 <div>
   <nav class="side-bar-nav">
-    <span class="fa fa-arrow-left hide-sidebar" ng-click="$storage.showSidebar = false" popover="Minimize sidebar" popover-placement="left" popover-trigger="mouseenter" popover-popup-delay="600"></span>
+    <div class="hide-sidebar-div">
+      <span class="fa fa-arrow-left hide-sidebar" ng-click="$storage.showSidebar = false" popover="Minimize sidebar" popover-placement="left" popover-trigger="mouseenter" popover-popup-delay="600"></span>
+    </div>
+    <div class="profile-pic">
+      <img src="img/jared.jpeg">
```

Comment:
whoooops, forgot to take that out. just added it to sample what an actual profile pic would look like.